Android/GoogleAndroidVending/docs/Expansion.md

```diff
@@ -0,0 +1,415 @@
+# APK Expansion Files
+
+[![NuGet](https://img.shields.io/nuget/vpre/Xamarin.Google.Android.Vending.Expansion.Downloader.svg)][4]
+[![NuGet](https://img.shields.io/nuget/dt/Xamarin.Google.Android.Vending.Expansion.Downloader.svg)][4]
```

Comment:
[1] not [4]

\[!\[NuGet](https://img.shields.io/nuget/vpre/Xamarin.Google.Android.Vending.Expansion.Downloader.svg)][1]
\[!\[NuGet](https://img.shields.io/nuget/dt/Xamarin.Google.Android.Vending.Expansion.Downloader.svg)][1]